### PR TITLE
Docs and SDK API example app improvements

### PR DESCRIPTION
**Pull request by johnpmitsch**
Some small docs enhancements. I didn't really understand the pagination at first, so I added a separate section explaining it.

I also added a `getNFTsByWalletAddress` API endpoint to the example app

### CODE REVIEW

File: packages/libs/api/sdk/README.md [hunk 1]
```diff
@@ -24,8 +24,6 @@ client.nft
   .then((response) => console.log(response));
 ```
 
-Full example implementation [here](https://github.com/user/repo/blob/branch/other_file.md).
```
Comment (AlejandroFabianCampos) [inline packages/libs/api/sdk/README.md:27]: Lol I hadn't noticed the link I left was broken, good catch!
that said, I think it'd still be good to point to the example app with a relative file link, wdyt?

**Review by AlejandroFabianCampos — APPROVED** [review]
Just this small suggestion, otherwise LGTM! Good work 🚀

File: packages/libs/api/sdk/README.md (hunk 1 above)
Comment (johnpmitsch) [reply packages/libs/api/sdk/README.md:27]: updated!

Comment (AlejandroFabianCampos) [reply packages/libs/api/sdk/README.md:27]: 🚢

**Pull request merged**